frontend/test/resolution/testResolve.cpp

```diff
@@ -1761,6 +1761,351 @@ static void test29(Context* context) {
   }
 }
 
+static void test30(Context* context) {
```

Comment:
Not a strong objection but IMO it would fit better in a `testIterators.cpp`. `testResolve.cpp` could contain almost everything from this directory but it shouldn't. We can keep it to "basic functionality" and "misc corner cases".